Core/DataObjects/PTMagicData.cs

```diff
@@ -48,7 +48,7 @@ public class Application
     public string TimezoneOffset { get; set; } = "+0:00";
     public string MainFiatCurrency { get; set; } = "USD";
     public string CoinMarketCapAPIKey { get; set; }
-    public string FreeCurrencyConverterAPIKey { get; set; }
+    public string FreeCurrencyConverterAPIKey { get; set; } = "sample-api-key";
```

Comment:
That should be left empty. There's no need for a sample key.

Reply:
This is the key they use on the website to demonstrate the API and it appears to work fine, surely this is ok as a default and folks can apply for a key and set it in the config if they want / need to?

Reply:
I still don't like the idea of PTM spamming their Sample API with requests.

Reply:
Yeah, it is a bit questionable... just wanted to eliminate yet another obstacle to getting the software running for users.

Shall I revert it then, or encourage users to sign up?

Reply:
Yeah. That's probably for the best.

Reply:
Ok I have added a commit to revert this change.